electrum/lnworker.py

```diff
@@ -1345,16 +1348,17 @@ def suggest_funding_amount(self, amount_to_pay, coins):
         min_funding_sat = max(min_funding_sat, 100_000) # at least 1mBTC
         if min_funding_sat > self.config.LIGHTNING_MAX_FUNDING_SAT:
             return
-        fee_est = partial(self.config.estimate_fee, allow_fallback_to_static_rates=True)  # to avoid NoDynamicFeeEstimates
+        #fee_est = partial(self.config.estimate_fee, allow_fallback_to_static_rates=True)  # to avoid NoDynamicFeeEstimates
+        fee_policy = FeePolicy(self.config.FEE_POLICY)
```

Comment:
soooo, what about NoDynamicFeeEstimates?
We could make `allow_fallback_to_static_rates` a field in `FeePolicy`. Then we could pass that to the constructor here.

Reply:
not done; this would not work everywhere. 
I think it is better to pass `allow_fallback_to_static_rates` as a parameter of `estimate_fee`
Note that a fallback to static feerates is still hard-coded for regtest, see FeePolicy.fee_per_kb

Reply:
note: before this commit, `allow_fallback_to_static_rates` was not passed to `mktx_for_open_channel` when called from `open_channel_with_peer`, because it used `fee_est=None` .